README.md

```diff
@@ -6,6 +6,16 @@ This buildpack installs **Google Chrome browser** `chrome` & [`chromedriver`](ht
 
 In summer 2023, the Chrome development team [addressed a long-standing problem with keeping Chrome & Chromedriver versions updated and aligned](https://developer.chrome.com/blog/chrome-for-testing/) with each other for automated testing environments. This buildpack follows this strategy to keep `chrome` & `chromedriver` versions  in-sync.
 
+## Installation
+
+⚠️ If migrating from a previous Chrome-chromedriver installation, then remove any pre-existing Chrome or Chromedriver buildpacks from the app.
```

Comment:
Maybe use a "> [!WARNING]" or similar instead of ⚠️? I don't think ⚠️ is super accessible. Other option here: https://github.com/orgs/community/discussions/16925